java/java.lsp.server/nbcode/nbproject/platform.properties

```diff
@@ -116,8 +116,6 @@ disabled.modules=\
     org.netbeans.modules.css.model,\
     org.netbeans.modules.css.prep,\
     org.netbeans.modules.css.visual,\
-    org.netbeans.modules.db.core,\
```

Comment:
Please check if those modules are actually needed - I couldn't find a reference to `db.core` or `db.dataview` APIs.